package.json

```diff
@@ -42,37 +42,39 @@
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/github": "^8.0.2",
     "@semantic-release/npm": "^8.0.3",
-    "@types/jest": "^27.0.2",
-    "@types/react": "^17.0.34",
-    "dotenv-cli": "^4.1.0",
-    "jest": "^27.3.1",
-    "jest-extended": "^1.1.0",
+    "@types/jest": "27.0.3",
```

Comment:
all carrets in the changed dependencies are removed, but not in the others, on purpose?

Reply:
Good catch!